lips/lip-1.mdx

```diff
@@ -11,149 +11,87 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-### Concurrency Control
-The Off-Chain Protocol manages concurrent requests on Objects and ensures that Object states will not become out of sync.
+**Shared Object**: All Objects contained within a Command are logically shared, meaning that each VASP has a copy of the Object and may create a new Command to modify it. For example, during the lifecycle of a `KycDataObject`, both VASPs will add information to it. The protocol does not support simultaneous updates to Objects. Rather, VASPs must take turns updating the Object.
 
-### Eventual Consistency
-The Off-Chain Protocol ensures eventual consistency of Shared Objects.
-
-# Basic Off-Chain Building Blocks
+**Reference ID**: Every Object type contains a `reference_id` field which is a unique reference ID for the Object. The reference ID is always specified by the payment initiator VASP (the VASP which originally created the Object). This value should be unique. We recommend the hexadecimal representation of a fresh >16 bytes random byte-string as a way of creating such values.
 
 ### On-Chain Account Setup
 A VASP's on-chain account must be created and set up in accordance with the standard VASP account creation process.
 
-Of particular note for off-chain APIs is the compliance key value.  This is the Ed25519 key with which a VASP signs travel rule statements on-chain and the required information within the KycDataObject exchanged via the Off-Chain Protocol.  During account creation, a VASP will set their compliance key which will be stored under the primary VASP account.
+Of particular note for off-chain APIs is the compliance key value.  This is the Ed25519 key with which a VASP signs travel rule statements on-chain and all off-chain requests and responses. During account creation, a VASP will set their compliance key which will be stored under the primary VASP account.
 
 ### HTTP end-point
 
-Each VASP exposes an HTTPS POST end point at `https://<hostname>:<port>/<protocol_version>/<LocalVASPAddress>/<RemoteVASPAddress>/command`.
-
-* `protocol_version` is initially 0 for this iteration of the off-chain APIs.
-* `LocalVASPAddress` and `RemoteVASPAddress` are the bech32 VASP addresses of the respective VASPs - with the subaddress set to 0's as per [LIP 5](https://lip.libra.org/lip-5/).
-    * An example URL: http://127.0.0.1:7091/v1/lbr1pg9q5zs2pg9q5zs2pg9q5zs2pgyqqqqqqqqqqqqqqspa3m/lbr1pgfpyysjzgfpyysjzgfpyysjzggqqqqqqqqqqqqqxjt54z/command/
-
+Each VASP exposes an HTTPS POST end point at `https://<hostname>:<port>/<protocol_version>/command`. The `protocol_version` is initially `v1` for this iteration of the off-chain APIs.
 
 ### HTTP Headers
-All HTTP requests and responses must contain a header `X-Request-ID` with a unique ID for the request, used for tracking requests and debugging. Responses must have the same string in the `X-Request-ID` header value as the requests they correspond to.
+All HTTP requests and responses must contain:
```

Comment:
let's be explicit here x-request-id and others are case-insensitive.

Reply:
Ok, I added: Both 

> `X-REQUEST-ID` and `X-VERIFICATION-KEY-ADRRESS` are case insensitive.